.github/workflows/build-workflow.yml

```diff
@@ -44,7 +44,14 @@ jobs:
     # 5. Build
     - name: Build with Gradle
       run: ./gradlew build
+
+    # 6. Upload APK Artifact
+    - name: Upload APK
+      uses: actions/upload-artifact@v1
+      with:
+        name: news-app
+        path: app/build/outputs/apk/debug/app-debug.apk
```

Comment:
Is it possible to have a wildcard here instead of absolute path?
Perhaps something like `*.apk`

Reply:
Okay. Let me check this. Thanks for suggestion.

Reply:
Can I use `app/**/*.apk`? I tested it and it's working.

Reply:
Interesting, maybe `/**/*.apk`?

Reply:
Actually `/` at the beginning means root path of Action's VM. That's why I choose `app/*`. 

See error:

```
  with:
    name: news-app
    path: /**/*.apk

##[error]EACCES: permission denied, scandir '/etc/polkit-1/localauthority'
```

Reply:
Ah okay. I guess then `app/**/*.apk` seems promising

Reply:
Finally I solved it using `./**/*.apk`. Let me remove unnecessary commits.

Reply:
Sweeet!